packages/zapp/console/src/components/Launch/LaunchForm/inputHelpers/map.ts

```diff
@@ -0,0 +1,81 @@
+import { stringifyValue } from 'common/utils';
+import { Core } from 'flyteidl';
+import { InputTypeDefinition, InputValue } from '../types';
+import { getHelperForInput } from './getHelperForInput';
+import { ConverterInput, InputHelper, InputValidatorParams } from './types';
+
+const missingSubTypeError = 'Unexpected missing subtype for map';
```

Comment:
All strings need to be added to strings file. example:
`packages/zapp/console/src/components/common/strings.ts`
usage:
{t('annotationsHeader')}

Reply:
fixed